viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -142,18 +149,18 @@ export class LighthouseReportViewer {
         this._reportIsFromGist = true;
         this._replaceReportHtml(reportJson);
       }).catch(err => logger.error(err.message));
-    } else if (jsonurl) {
-      const firebaseAuth = this._github.getFirebaseAuth();
-      loadPromise = firebaseAuth.getAccessTokenIfLoggedIn()
-        .then(token => {
-          return token
-            ? Promise.reject(new Error('Can only use jsonurl when not logged in'))
-            : null;
-        })
-        .then(() => fetch(jsonurl))
+    } else if (jsonurl ?? cafeid) {
```

Comment:
nit: this is the effective condition so let's make it explicit
```suggestion
    } else if (jsonurl || cafeid) {
```